app/models/node.rb

```diff
@@ -871,7 +878,6 @@ def add_tag(tagname, user)
 
             if node_tag.save
               saved = true
-              tag.run_count # update count of tag usage
```

Comment:
it is already updated with a callback in the node_tag.rb file 

https://github.com/publiclab/plots2/blob/a29514a4378d2eed2ff99167153da3becf915d24/app/models/node_tag.rb#L11